site/content/docs/5.2/forms/validation.md

```diff
@@ -169,6 +184,13 @@ For invalid fields, ensure that the invalid feedback/error message is associated
 
 To fix [issues with border radius](https://github.com/twbs/bootstrap/issues/25110), input groups require an additional `.has-validation` class.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
I suppose because of the horizontal layout. Should be mentioned like this everywhere when it happens IMO.

Reply:
This callout can be deleted. The only thing that needs to be corrected (issue #1715) is to make the checkbox label a link toward the text of the terms and conditions to accept. This rule will be added in the DSM later. The fix will need a design review.

Reply:
Resolved ?